### PR TITLE
HAL-1607: Configure HotRod session manager in distributable-web subsystem

### PR DESCRIPTION
**Pull request by hpehl**
https://issues.jboss.org/browse/HAL-1607

### CODE REVIEW

**Comment by zhantemirov** [comment]
@hpehl Hallo,

as I understood there is no much to write in the Analysis Document about the feature request (I have seen your previous PRs with similar feature requests). Should I consider this AD state as a final one?

Sultan.

**Comment by hpehl** [comment]
Yes it's final.

**Comment by tommaso-borgato** [comment]
@hpehl sorry for asking but I am not familiar with Web Console RFEs: who is going to write the tests here?

**Comment by hpehl** [comment]
@tommaso-borgato that's described in the AD which is attached to the relevant EAP7 issue. In that case it's done by QE.

**Comment by tommaso-borgato** [comment]
@hpehl can you point me at some examples, info, etc. on how to write those kind of test?

**Comment by hpehl** [comment]
@tommaso-borgato the test suite is here: https://github.com/hal/testsuite.next. 

It's based on on Drone & Graphene Arquillian extensions. There are some simple tests like which you can take a look at. Almost all tests follow these guidelines:

1. Prepare the model
1. Open a specific page
1. Simulate user input
1. Assert results
1. Cleanup

File: console/HAL-1607_Configure_HotRod_distributable-web_subsystem.adoc [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+=== QE Contacts
+
+* mailto:szhantem@redhat.com[Sultan Zhantemirov]
+
+=== Testing By
+    
+[ ] Engineering
+    
+[x] QE
```
Comment (mnovak1) [inline console/HAL-1607_Configure_HotRod_distributable-web_subsystem.adoc]: Should be switched to tested by `Engineering`.

**Pull request merged**